schemas/notes.py

```diff
@@ -0,0 +1,29 @@
+from ma import ma
+from models.notes import NotesModel
+from models.user import UserModel
+from utils.time import time_format
+from marshmallow import fields, validates, ValidationError
+
+class NotesSchema(ma.SQLAlchemyAutoSchema):
+    class Meta:
+        model = NotesModel
+        include_fk = True
+        include_relationships = True
+        exclude = ["userinfo"]
```

Comment:
I believe this only affects serialization. I would prefer to leave this out since we can specify exclude parameters at initiation of the schema. Of course that is just my preference. We could discuss this more too.

Reply:
You are correct. the exclude only affects serialization, and can be specified when creating a schema object. Leaving it out does make it more flexible.